pkgs/applications/networking/browsers/vivaldi/ffmpeg-codecs.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv, fetchurl, fetchpatch
+, dbus_glib, gtk2, gtk3, libexif, libpulseaudio, libXScrnSaver, ninja, nss
+, pciutils, pkgconfig, python2, xdg_utils
+}:
+
+stdenv.mkDerivation rec {
+  name = "${product}-${version}";
+  product = "vivaldi-ffmpeg-codecs";
+  version = "59.0.3071.104";
```

Comment:
Circular dependency is solvable, at least there is a trivial solution - put them into a single project.

The versions have good chance to diverge.

Browsers are updated often, usually in few hours after a new version is released, and those updates are make each time by different people. I do not expect all of the updaters would notice existence of ```vivaldi-ffmpeg-codecs``` and get an idea to run vivaldi, open vivaldi://about/ and update chrome version in ```vivaldi-ffmpeg-codecs```.